contracts/deploy/03-routers/03-arb-to-gnosis-router.ts

```diff
@@ -39,11 +39,30 @@ const deployRouter: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const hardhatDeployer = async () => {
     const veaOutbox = await deployments.get("VeaOutboxArbToGnosis");
     const veaInbox = await deployments.get("VeaInboxArbToGnosis");
+    const amb = await deployments.get("MockAMB");
+    //const ArbSysMock = await deployments.get('arbSysMock');
+
+    const sequencerInbox = await deploy("SequencerInboxMock", {
```

Comment:
This deploy call and the one under it could benefit from concurrency

Reply:
already answered